tox.ini

```diff
@@ -32,6 +32,7 @@ commands = check-manifest
 show-source = true
 max-line-length = 80
 exclude = .venv,.tox,.git,dist,doc,*.egg,build
+ignore = E221
```

Comment:
This is for the whitespace between attributes and definition in `GCPResourceRecordSet`. 

FYI the default Sublime Text Python Flake8 Lint package picks this up and overrides any global flake8 config you may have. In the user settings config, set `use_flake8_project_config` to `False` if this bugs you!

Reply:
Yeah then they make the line longer than 80 chars 😭  not sure if there's a module-wide 

I can just remove the spacing in `GCPResourceRecordSet` too; after I pushed this, I felt it'd be kind of inconsistent in style, anyways. I'm going to leave it to you @matt0bi to make the call :)

Reply:
Removing the spacing seems like the simplest option, the class will still be super readable :).